src/config.ts

```diff
@@ -34,30 +32,25 @@ const createConfig = (env): Config => {
   };
 };
 
-const config = createConfig({});
-
-export const getConfig = (): Config => config;
-
-export const loadConfig = async (): Promise<void> => {
-  const response = await axios.get('/env.json');
-  const env = response.data;
-
-  // override all env variables to ut1
-  // Object.assign(env, {
-  //   REGISTRATION_HOST: 'https://registrering.ut1.fellesdatakatalog.brreg.no',
-  //   CONCEPT_REGISTRATION_API: 'https://registrering-begrep-api.ut1.fellesdatakatalog.brreg.no',
-  //   SSO_HOST: 'https://sso.ut1.fellesdatakatalog.brreg.no',
-  //   PUBLISHER_API: 'https://www.ut1.fellesdatakatalog.brreg.no',
-  //   PUBLISHER_DATA_AUTHORIZATION: 'Basic ZmRrOkJSUkVH'
-  // });
+const env = (window as any).env || {
+  // use docker
+  CONCEPT_REGISTRATION_API: 'http://localhost:8200',
+  REGISTRATION_HOST: 'https://localhost:8098',
+  PUBLISHER_API: 'http://localhost:8080',
+  PUBLISHER_DATA_AUTHORIZATION: '',
+  // PUBLISHER_API:'https://www.ut1.fellesdatakatalog.brreg.no'
+  // PUBLISHER_DATA_AUTHORIZATION:'Basic ZmRrOkJSUkVH' // for ut1 and st1
+  SSO_HOST: 'http://localhost:8084'
+};
 
-  // override all env variables to local docker
-  // Object.assign(env, {
-  //   REGISTRATION_HOST:'http://localhost:8098',
-  //   CONCEPT_REGISTRATION_API:'http://localhost:8200',
-  //   SSO_HOST:'http://localhost:8084',
-  //   PUBLISHER_API:'http://localhost:8080',
-  // });
+// use ut1
+// || {
+//   REGISTRATION_HOST: 'https://registrering.ut1.fellesdatakatalog.brreg.no',
+//   CONCEPT_REGISTRATION_API: 'https://registrering-begrep-api.ut1.fellesdatakatalog.brreg.no',
+//   SSO_HOST: 'https://sso.ut1.fellesdatakatalog.brreg.no',
+//   PUBLISHER_API: 'https://www.ut1.fellesdatakatalog.brreg.no',
+//   PUBLISHER_DATA_AUTHORIZATION: 'Basic ZmRrOkJSUkVH'
+// }
+const config = createConfig(env);
 
-  Object.assign(config, createConfig(env));
-};
+export const getConfig = (): Config => config;
```

Comment:
Does not need to be a function anymore, but it may be a more extensive refactoring for another time